src/Cmfd.cpp

```diff
@@ -880,6 +886,10 @@ void Cmfd::initializeMaterials() {
 
   Material* material;
 
+  /* Delete old Cmfd surface currents vector if it exists */
+  if (_materials != NULL)
+    delete _materials;
```

Comment:
I think this should be `delete [] _materials`, otherwise I would assume just the pointer to the array is deleted and not the actual array.